src/main/webapp/dev/js/common/instructor.js

```diff
@@ -358,14 +358,19 @@ function bindCourseDeleteAllLinks() {
         event.preventDefault();
 
         const $clickedLink = $(event.currentTarget);
-        const messageText = 'Are you sure you want to permanently delete all the courses? '
-                + 'This operation will delete all students and sessions in these courses. '
-                + 'All instructors of these courses will not be able to access them hereafter as well.';
+        const $softDeletedCourses = $('td[id^="recoverycourseid"]');
+        let messageText = `Are you sure you want to permanently delete all ${$softDeletedCourses.length} courses in `
+                + 'the <b>Recycle Bin</b>? This operation will delete all students and sessions in these courses. '
+                + 'All instructors of these courses will not be able to access them hereafter as well.<br><br>'
```

Comment:
Is there any reason they can't be split up by paragraphs? Two `br` elements are very awkward.